packages/core-utils/src/query-params.js

```diff
@@ -365,6 +365,30 @@ const queryParams = [
       }
     ]
   },
+  {
+    /* walkSpeed -- the user's walking speed in m/s */
+    name: "walkSpeed",
+    routingTypes: ["ITINERARY", "PROFILE"],
+    default: 1.34,
+    selector: "DROPDOWN",
+    label: "Walk Speed",
+    applicable: query =>
+      !query.otp2 && query.mode && query.mode.indexOf("WALK") !== -1,
+    options: [
+      {
+        text: "2 MPH",
+        value: 0.89
+      },
+      {
+        text: "3 MPH",
+        value: 1.34
+      },
+      {
+        text: "4 MPH",
+        value: 1.79
+      }
+    ]
+  },
```

Comment:
hi @daniel-heppner-ibigroup 👋 , I saw that you removed this section of the code last August in [this commit](https://github.com/opentripplanner/otp-ui/commit/2438812f059a325d3f25b030f32d993d4250b6ee).  At TriMet we're still on OTP 1 for now and this UI element is still in the current version of our app so I'm hoping to add it back to the `TripOptions` component only, but want to make sure it doesn't break anything elsewhere in the code base.  

I added some logic to the `applicable` method to exclude it if a query param like `otp2: true` is passed, but not sure if that addresses the reason it was removed.  I also confirmed that it doesn't show up in the `SettingSelectorPanel` with these additions (at least in the Storybook stories for that component)

Reply:
per Miles request I'm now overriding the version of `core-utils` that `trip-form` uses in the trimet.org app to go back to 9.0.3, the last that had the `walkSpeed` param.  So this is now a moot point and there are no longer changes to `core-utils` in this PR